Makefile

```diff
@@ -30,6 +30,20 @@ test-e2e:
 test-all:
 	clojure -M:test
 
+test-common:
+	clojure -M:test-common
+
+create-common-test:
```

Comment:
Goed idee om automatisch alle files te laden Wel nog een paar opmerkingen over `make`:

- Je genereert hier `test-common/nl/surf/eduhub_rio_mapper/dependency_test.clj` -- dat is dan ook de target van de rule.
- De dependencies van de rule zijn dan ook de source files in `src-common` -- deze wil je dan ook specificeren in de makefile, maar dat is wel tricky: zie https://www.gnu.org/software/make/manual/html_node/Automatic-Prerequisites.html

Als alternatief zou je het scannen en inladen van  src-common files ook helemaal in clojure kunnen doen, dan heb je een "gewone" test file en is denk ik uiteindelijk leesbaarder en betrouwbaarder.